### PR TITLE
Add AI4Code Kaggle jupyter notebook code

### PR DESCRIPTION
**Pull request by PhungVanDuy**

### CODE REVIEW

File: codepile/ai4code_kaggle/ai4code_kaggle.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import os
+import gdown
+import pandas as pd
+from codepile.dataset import Dataset
+
+AI4CODE_KAGGLE_URL = "https://drive.google.com/file/d/1RgiY7_zvLPdMPOXaOL4aE-Or5KuZXQ02/view?usp=sharing"
```
Comment (reshinthadithyan) [inline codepile/ai4code_kaggle/ai4code_kaggle.py]: GDrive links are volatile. Is this your own private drive link? @PhungVanDuy

Comment (PhungVanDuy) [reply codepile/ai4code_kaggle/ai4code_kaggle.py]: Yes I understand, it's totally temporary, I am waiting for @ncoop57 set up an S3 bucket to store all datasets.

**Comment by ncoop57** [comment]
@PhungVanDuy could you enable maintainer edits? https://github.blog/2016-09-07-improving-collaboration-with-forks/

**Comment by PhungVanDuy** [comment]
@ncoop57  I just added you to my repo, you should have all permission :) Thanks!

**Comment by PhungVanDuy** [comment]
@ncoop57  can you review this one also ?

**Comment by ncoop57** [comment]
LGTM

**Pull request merged**